### DISCUSSION TITLE
How enable / add icons to directories like src, middlewares, routes, etc?

### DISCUSSION THREAD

**Discussion by monoira** [LazyVim/LazyVim#5098] [Q&A] [answered]
![Screenshot from 2024-12-15 21-02-25](https://github.com/user-attachments/assets/9c2f34c1-a8ec-43a6-b2ae-a9c2de41d2aa)
![Screenshot from 2024-12-15 21-04-46](https://github.com/user-attachments/assets/f13cd179-18a8-44af-bd25-585a491ab706)

**Comment by DrummyFloyd** [comment, marked as answer]
try to use this  https://github.com/echasnovski/mini.icons?tab=readme-ov-file#default-config

```
return {
  "echasnovski/mini.icons",
  opts = {
    directory = {
      middlewares = { glyph = "<YOUICON>" },
    },
  },
}

 ```

**Reply by monoira** [reply]
Thanks for the answer.
I was wondering if there was another way beyond giving each directory it's own icon.
I'll wait for such answer and if I can't find one, then I will Mark your answer as The answer and move on.

EDIT: nvm. It is what it is.
